joins/operations/default.json

```diff
@@ -0,0 +1,243 @@
+    {
+      "name": "index-base",
+      "operation-type": "bulk",
+      "indices": [
+        "join_base_idx"
+      ],
+      "bulk-size": {{bulk_size | default(10000)}},
+      "ingest-percentage": {{ingest_percentage | default(100)}}
+    },
+    {
+      "name": "index-small-lookup-indices",
+      "operation-type": "bulk",
+      "indices": [
+        "lookup_idx_1000_f10",
+        "lookup_idx_100000_f10",
+        "lookup_idx_200000_f10",
+        "lookup_idx_500000_f10"
+      ],
+      "bulk-size": {{bulk_size | default(10000)}},
+      "ingest-percentage": 100
+    },
+    {
+      "name": "index-lookup-1m",
+      "operation-type": "bulk",
+      "indices": ["lookup_idx_1000000_f10"],
+      "bulk-size": {{bulk_size | default(10000)}},
+      "ingest-percentage": {{ingest_percentage | default(100)}}
+    },
+    {
+      "name": "index-lookup-5m",
+      "operation-type": "bulk",
+      "indices": ["lookup_idx_5000000_f10"],
+      "bulk-size": {{bulk_size | default(10000)}},
+      "ingest-percentage": {{ingest_percentage | default(100)}}
+    },
+    {
+      "name": "index-lookup-100m",
+      "operation-type": "bulk",
+      "indices": ["lookup_idx_100000000_f10"],
+      "bulk-size": {{bulk_size | default(10000)}},
+      "ingest-percentage": {{ingest_percentage | default(100)}}
+    },
+
+    {
+      "name": "esql_lookup_join_1k_keys_limit1",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_1000_f10 on key_1000 | limit 1"
+    },
+    {
+      "name": "esql_lookup_join_1k_keys_limit1000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_1000_f10 on key_1000 | limit 1000"
+    },
+    {
+      "name": "esql_lookup_join_1k_keys_limit10000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_1000_f10 on key_1000 | limit 10000"
+    },
+    {
+      "name": "esql_lookup_join_1k_keys_where_limit1000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_1000_f10 on key_1000 | where lookup_keyword_0 like \"val 1*\" | limit 1000"
+    },
+    {
+      "name": "esql_lookup_join_1k_keys_where_no_match",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_1000_f10 on key_1000 | where concat(lookup_keyword_0, \"foo\") == \"bar\""
```

Comment:
For now these queries work the same as the `*keys_where_limit*` (see above), but in the future we could be able to push down to lucene one of the two, but most likely not the other one.